nx-X11/programs/Xserver/Xext/shm.c

```diff
@@ -167,7 +166,7 @@ static Bool CheckForShmSyscall()
     int shmid = -1;
 
     /* If no SHM support in the kernel, the bad syscall will generate SIGSYS */
-    oldHandler = signal(SIGSYS, SigSysHandler);
+    oldHandler = OsSignal(SIGSYS, SigSysHandler);
```

Comment:
@Ionic:

> What about nx-X11/programs/Xserver/os/utils.c? Missing changes for SIGCHLD there.

Oh dear, this commit has been completely redone. Basically by pulling in all recent and ancient changes to the smart scheduler ABI.